packages/stark-ui/src/modules/table/components/column.component.html

```diff
@@ -34,7 +34,8 @@
 	</th>
 	<!-- the column template defined by the user will be displayed here -->
 	<!-- and it will receive the right context containing the displayedValue and the row data-->
-	<td mat-cell *matCellDef="let rowItem" [ngClass]="getCellClassNames(rowItem)" (click)="onCellClick($event, rowItem)">
+	<td mat-cell *matCellDef="let rowItem" [ngClass]="getCellClassNames(rowItem)" (click)="onCellClick($event, rowItem)" 
+		[matTooltip]="getCellTooltip(rowItem)" [matTooltipPosition]="cellTooltipPosition" [matTooltipClass]="getCellTooltipClassNames(rowItem)">
```

Comment:
What if the tooltip is an empty string `""`? I think we should disable the tooltip in such cases with the `matTooltipDisabled` property otherwise I fear that Angular Material would still be doing some logic to display the tooltip which will be overkill if that would be an empty string in the end.

Another solution would be to have 2 `<td mat-cell>` with an `*ngIf`:  one will contain the `[matTooltip]` if there is a relevant tooltip to show (not empty). And the other one won't contain the `[matTooltip]`, basically the same we had before, when there is no tooltip to show.

What do you think?